src/server/middleware/request-security.middleware.ts

```
@@ -0,0 +1,236 @@
+  }
+
+  /**
+   * Apply timeout protection to a response
+   */
+  applyTimeoutProtection(
+    req: IncomingMessage,
+    res: ServerResponse,
+    requestLogger: Logger,
+  ): { cleanup: () => void } {
+    let requestCompleted = false;
+    
```

ESLint

[error] Delete `····`

(prettier/prettier)